### DISCUSSION TITLE
C# support on Android in Godot 4

### DISCUSSION THREAD

**Discussion by Hamster5295** [godotengine/godot-proposals#6972] [Platforms]
It was claimed in [this passage](https://godotengine.org/article/whats-new-in-csharp-for-godot-4-0/) that C# was not yet supported on mobile platform when Godot 4 came out.  
I've been working for something else since that, and have totally lost track of the dev process. Does anyone get the idea whether C# is supported now, or whether there's any progress on the implementation? 🐱

**Comment by AThousandShips** [comment]
Already tracked in godotengine/godot#68153

**Reply by Hamster5295** [reply]
Get it, thank you! 😄
